app/assets/javascripts/app/controllers/_profile/email_header.coffee

```diff
@@ -0,0 +1,87 @@
+class Index extends App.ControllerSubContent
+  requiredPermission: 'user_preferences.email_header'
```

Comment:
I think there is no need to limit this functionality to a certain permission since it's just a matter of taste. JFI: Permissions need to get registered: https://github.com/zammad/zammad/blob/develop/db/seeds/permissions.rb